lib/cb/clients/base.rb

```diff
@@ -14,6 +14,14 @@ class Base
       def self.cb_client
         @cb_client ||= Cb::Utils::Api.instance
       end
+
+      def self.headers(args)
+        {
+            'Accept' => 'application/json',
+            'Authorization' => "Bearer #{ args[:oauth_token] }",
```

Comment:
All APIs (user centric and client credential flow) should be able to work this way.  If I use a user centric token then Michelle's layer will append user did etc on the other end.  If I use a client credential token she won't.